### DISCUSSION TITLE
License Handling?

### DISCUSSION THREAD

**Discussion by raydeo** [jgraph/drawio#2000] [General]
Hi,

Please can you tell me if.

Draw.io falls under FOSS/FOSSA?

Draw.io under GNU or Mozilla or Eclipse license?

Many thanks for the confirmation!

**Comment by CodingLama** [comment]
That's more like a question for a layer.

**Comment by davidjgraph** [comment]
There is a LICENSE file and a LICENSE section in the root of this repo.
